operatorconfig/moduleconfig/common/version-values.yaml

```diff
@@ -13,7 +13,7 @@ powerscale:
   v2.8.0:
       authorization: "v1.7.0"
       replication: "v1.5.0"
-      observability: "v1.5.0"
+      observability: "v1.6.0"
       resiliency: "v1.7.0"
 powerflex:
   # List of Driver versions and modules that supports the version
```

Comment:
Not in the scope of this PR but please help remove entries related to 2.5.0 for powerflex in this file

Reply:
Removed.